tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -2842,6 +2844,53 @@ public function test_allow_tracking_disabled() {
 		$this->assertTrue( apply_filters( 'googlesitekit_allow_tracking_disabled', true ) );
 	}
 
+	public function test_get_custom_dimensions_data() {
+		global $wp_query;
+
+		$settings = array(
+			'availableCustomDimensions' => array(
+				'googlesitekit_post_author',
+				'googlesitekit_post_type',
```

Comment:
It would be nice to also add test coverage for the `googlesitekit_post_categories` and `googlesitekit_post_date` dimensions...

Reply:
Yep, added.